web/src/app/users/UserContactMethodList.tsx

```diff
@@ -102,6 +104,9 @@ export default function UserContactMethodList(
       },
     ]
 
+    // don't show send test for slack DMs if disabled
+    if (cm.type === 'SLACK_DM' && !hasSlackDM) return actions
```

Comment:
is the experimental flag needed here? could probably leave it out here

Reply:
If the flag is turned off after a user has added a Slack contact method, we don't want to show the Test/Verify actions because they will be broken; otherwise, it will generate failures/errors in the backend every time anyone tries to use them.